src/pages/components/content-block-cards.mdx

```diff
@@ -25,8 +25,9 @@ import ResourceLinks from 'components/ResourceLinks';
 
 ## Overview
 
-By default the Content block cards simply supports only text for moments needing richer content, and sets proper
-expectations for the user to make their next journey decision from within the page's narrative.
+The default version pairs the Content block heading and [card group](https://www.ibm.com/standards/carbon/components/card-group) component. Use when you need to introduce a card group as its own content block without a description.
```

Comment:
Small question regarding capitalization here -- in `Content block` the `c` is capitalized, but not the `c` in `card group`. Should only the first component be capitalized in a sentence?

Reply:
Oh man @IgnacioBecerra idk how I did this, but this PR should have been only the video page!